docs/book/src/user/troubleshooting.md

```diff
@@ -36,4 +36,17 @@ For convenience, here is an example one-liner to do this post installation
 
 ```
 kubectl get nodes --no-headers -l '!node-role.kubernetes.io/master' -o jsonpath='{range .items[*]}{.metadata.name}{"\n"}' | xargs -I{} kubectl label node {} node-role.kubernetes.io/worker=''
-```
+```                  
+
+## Cluster API with Docker
+
+When provisioning workload clusters using Cluster API with Docker infrastructure,
+provisioning might be stuck: 
+ 
+1. if there are stopped containers on your machine from previous runs. Clean unused containers with [docker rm -f ](https://docs.docker.com/engine/reference/commandline/rm/). 
+
+2. if the docker space on your disk is being exhausted 
+    * Run [docker system df](https://docs.docker.com/engine/reference/commandline/system_df/) to inspect the disk space consumed by Docker resources.
+    * Run [docker volume prune](https://docs.docker.com/engine/reference/commandline/volume_prune/) to prune volumes that are not used by at least one container.
```

Comment:
Let's suggest to use `docker system prune --volumes` because it will get rid of dangling images, containers, volumes and networks

Reply:
It seems the command is not yet as expected. It should be 

`docker system prune --volumes`

instead of

`docker volume prune --volumes`